### PR TITLE
Post kotlin-dsl 0.19.0 activities

### PR DESCRIPTION
**Pull request by eskatos**
Follow up to #6071

### CODE REVIEW

**Comment by eskatos** [comment]
Feedback build https://builds.gradle.org/viewQueued.html?itemId=14480041

File: buildSrc/build.gradle.kts [hunk 1]
```diff
@@ -187,12 +189,17 @@ fun Project.applyGroovyProjectConventions() {
...
-    //apply(plugin = "org.gradle.kotlin.ktlint-convention")
+    apply(plugin = "org.gradle.kotlin.ktlint-convention")
 
     tasks.withType<KotlinCompile>().configureEach {
         kotlinOptions {
             freeCompilerArgs += listOf("-Xjsr305=strict")
         }
     }
+
+    plugins.withType<KotlinDslPlugin> {
```
Comment (bamboo) [inline buildSrc/build.gradle.kts]: ❓ `.withType<KotlinDslPlugin>().configureEach {`?

Comment (eskatos) [reply buildSrc/build.gradle.kts]: `plugins` elements are realized as soon as they are applied, so I don't think that's necessary

**Comment by eskatos** [comment]
New feedback build https://builds.gradle.org/viewQueued.html?itemId=14488340

**Pull request merged**